spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/rdd/CassandraReader.scala

```diff
@@ -0,0 +1,166 @@
+package com.datastax.spark.connector.rdd
+
+import java.io.IOException
+
+import com.datastax.driver.core.{ProtocolVersion, Session}
+import com.datastax.spark.connector._
+import com.datastax.spark.connector.cql.{CassandraConnector, Schema}
+import com.datastax.spark.connector.rdd.reader._
+
+import scala.reflect.ClassTag
+
+trait CassandraReader[R] {
```

Comment:
Yeah it is mostly the code to generate the RowReader for the given CassandraRDD, So maybe "CassandraRowReader[R]" This way it is written Class with CassandraRowReader[R]